datafusion/sql/src/planner.rs

```diff
@@ -3837,10 +3759,9 @@ mod tests {
             LEFT JOIN orders \
             ON id = customer_id AND order_id > 1 AND age < 30";
         let expected = "Projection: #person.id, #orders.order_id\
-        \n  Left Join: #person.id = #orders.customer_id Filter: #person.age < Int64(30)\
+        \n  Left Join: #person.id = #orders.customer_id Filter: #orders.order_id > Int64(1) AND #person.age < Int64(30)\
         \n    TableScan: person projection=None\
-        \n    Filter: #orders.order_id > Int64(1)\
-        \n      TableScan: orders projection=None";
+        \n    TableScan: orders projection=None";
```

Comment:
it seems that it can't pushdown filter to TableScan👀

Reply:
You're right, it can not anymore - now all predicate movements are on `filter_push_down` optimizer, and there is no need for planner to define should it push some filters down or not - it just creates plan node as it is.

Reply:
So to be clear, when run normally the pushdown will still happen, but they are no longer pushed down by the planner, rather they are pushed down at a later stage.

👍